src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java

```diff
@@ -158,6 +170,16 @@ public static ListingType getListingTypeFromCaseData(ListingData listingData, Ca
         }
     }
 
+    private static String checkScotlandReadingDeliberationDay(DateListedType dateListedType, CaseData caseData) {
+        if (caseData.getManagingOffice() == null
+            || isNullOrEmpty(dateListedType.getHearingTypeReadingDeliberation())
+            || dateListedType.getHearingTypeReadingDeliberation().equals("Neither")) {
```

Comment:
The two Ors can be combined by this: "Neither".equals(dateListedType.getHearingTypeReadingDeliberation())

Reply:
There's no need for the managingOffice check either really. And why not just call it getScotland....

Reply:
Refactored